kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -271,6 +274,18 @@ public static ConfigDef newConfigDef() {
           "S3 Object Tagging"
       );
 
+      configDef.define(
+              S3_OBJECT_IGNORE_TAGGING_ERROR_CONFIG,
+              Type.BOOLEAN,
```

Comment:
This could be modelled as behavior.on.tagging.error (along the lines of behavior.on.null.values) with 2 options ignore, and fail.